============================= test session starts ==============================
platform linux -- Python 3.10.12, pytest-7.4.4, pluggy-1.4.0
rootdir: /home/khashayar/projs/det-gen
plugins: anyio-4.3.0
collected 3 items

c4b_ret_list_bak/src_16/0/test_transformed_src_16.py .F.                 [100%]

=================================== FAILURES ===================================
_________________________________ test_case_2 __________________________________

    def test_case_2():
        bytes_0 = b"\\k\xef\x9ae"
        var_0 = module_0.func(*bytes_0)
        bool_0 = True
        list_0 = module_0.func(*bytes_0)
>       var_1 = module_0.func(*list_0)
E       TypeError: src_16.func() argument after * must be an iterable, not NoneType

c4b_ret_list_bak/src_16/0/test_transformed_src_16.py:27: TypeError
=========================== short test summary info ============================
FAILED c4b_ret_list_bak/src_16/0/test_transformed_src_16.py::test_case_2 - Ty...
========================= 1 failed, 2 passed in 0.07s ==========================
